json/access/src/test/java/ch/qos/logback/contrib/json/access/JsonLayoutTest.java

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright (C) 2016, The logback-contrib developers. All rights reserved.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation
+ *
+ *   or (per the licensee's choosing)
+ *
+ * under the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation.
+ */
+package ch.qos.logback.contrib.json.access;
+
+import ch.qos.logback.access.TeztConstants;
+import ch.qos.logback.access.dummy.DummyAccessEventBuilder;
+import ch.qos.logback.access.joran.JoranConfigurator;
+import ch.qos.logback.access.spi.AccessContext;
+import ch.qos.logback.access.spi.IAccessEvent;
+import ch.qos.logback.core.joran.spi.JoranException;
+import ch.qos.logback.core.read.ListAppender;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import static ch.qos.logback.contrib.json.access.JsonLayout.REQUESTTIME_ATTR_NAME;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class JsonLayoutTest {
+
+    private AccessContext context = new AccessContext();
+
+    private void configure(String file) throws JoranException {
+        JoranConfigurator jc = new JoranConfigurator();
+        jc.setContext(context);
+        jc.doConfigure(file);
+    }
+
+    @Test
+    public void addToJsonMap() throws Exception {
+        Map<String, Object> map = new LinkedHashMap<String, Object>();
+        JsonLayout jsonLayout = new JsonLayout();
+        jsonLayout.add("key1", true, "value1", map);
+        jsonLayout.add("key2", false, "value2", map);
+        jsonLayout.add("key3", true, null, map);
+
+        assertTrue(map.size() == 1);
+        assertTrue(map.containsKey("key1"));
+        assertEquals(map.get("key1"), "value1");
+        assertTrue(!map.containsKey("key2"));
+        assertTrue(!map.containsKey("key3"));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [assertTrue(!expr) can be replaced by assertFalse(expr)](https://www.codacy.com/app/tony19/logback-contrib/file/2377134886/issues/source?bid=3222643&fileBranchId=3222645#l55)